### ISSUE TITLE
Rollup failed to resolve import "{}" from "node_modules/@storybook/client-logger/dist/esm/index.js"

### ISSUE THREAD

**Issue by derekr** [eirslett/storybook-builder-vite#71]
Hey! Having issues trying to build storybook. I recreated the issue in a stackblitz:
https://stackblitz.com/edit/storybook-vite-builder-client-logger-build-bug?file=package.json

Running storybook in dev works OK locally (stackblitz seems to be choking on installing esbuild), but the `npm run build-storybook` command errors out the same way I'm experiencing locally and hope it's enough of a clue.

```sh
[vite]: Rollup failed to resolve import "{}" from "node_modules/@storybook/client-logger/dist/esm/index.js".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`
ERR! Error: [vite]: Rollup failed to resolve import "{}" from "node_modules/@storybook/client-logger/dist/esm/index.js".
ERR! This is most likely unintended because it can break your application at runtime.
ERR! If you do want to externalize this module explicitly add it to
ERR! `build.rollupOptions.external`
```

<details>
<summary>Full command output</summary>

```sh
❯ npm run build-storybook
$ build-storybook -s public
info @storybook/react v6.3.4
info 
info => Cleaning outputDir: /home/projects/storybook-vite-builder-client-logger-build-bug/storybook-static
info => Copying static files: ./public => ./
info => Loading presets
info => Compiling manager..
vite v2.4.2 building for production...
✓ 15 modules transformed.
[vite]: Rollup failed to resolve import "{}" from "node_modules/@storybook/client-logger/dist/esm/index.js".
This is most likely unintended because it can break your application at runtime.
If you do want to externalize this module explicitly add it to
`build.rollupOptions.external`
ERR! Error: [vite]: Rollup failed to resolve import "{}" from "node_modules/@storybook/client-logger/dist/esm/index.js".
ERR! This is most likely unintended because it can break your application at runtime.
ERR! If you do want to externalize this module explicitly add it to
ERR! `build.rollupOptions.external`
ERR!     at onRollupWarning (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/vite/dist/node/chunks/dep-11db14da.js:51691:19)
ERR!     at onwarn (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/vite/dist/node/chunks/dep-11db14da.js:51481:13)
ERR!     at Object.eval [as onwarn] (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/rollup/dist/shared/rollup.js:20470:13)
ERR!     at ModuleLoader.handleResolveId (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/rollup/dist/shared/rollup.js:19819:26)
ERR!     at eval (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/rollup/dist/shared/rollup.js:19766:22)
ERR!     at async Promise.all (index 8)
ERR!     at async ModuleLoader.fetchStaticDependencies (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/rollup/dist/shared/rollup.js:19764:34)
ERR!     at async Promise.all (index 0)
ERR!     at async ModuleLoader.fetchModule (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/rollup/dist/shared/rollup.js:19740:9)
ERR!     at async Promise.all (index 2)
ERR!     at async ModuleLoader.fetchStaticDependencies (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/rollup/dist/shared/rollup.js:19764:34)
ERR!     at async Promise.all (index 0)
ERR!     at async ModuleLoader.fetchModule (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/rollup/dist/shared/rollup.js:19740:9)
ERR!     at async Promise.all (index 0)
ERR!     at async ModuleLoader.fetchStaticDependencies (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/rollup/dist/shared/rollup.js:19764:34)
ERR!     at async Promise.all (index 0)
ERR!  Error: [vite]: Rollup failed to resolve import "{}" from "node_modules/@storybook/client-logger/dist/esm/index.js".
ERR! This is most likely unintended because it can break your application at runtime.
ERR! If you do want to externalize this module explicitly add it to
ERR! `build.rollupOptions.external`
ERR!     at onRollupWarning (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/vite/dist/node/chunks/dep-11db14da.js:51691:19)
ERR!     at onwarn (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/vite/dist/node/chunks/dep-11db14da.js:51481:13)
ERR!     at Object.eval [as onwarn] (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/rollup/dist/shared/rollup.js:20470:13)
ERR!     at ModuleLoader.handleResolveId (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/rollup/dist/shared/rollup.js:19819:26)
ERR!     at eval (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/rollup/dist/shared/rollup.js:19766:22)
ERR!     at async Promise.all (index 8)
ERR!     at async ModuleLoader.fetchStaticDependencies (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/rollup/dist/shared/rollup.js:19764:34)
ERR!     at async Promise.all (index 0)
ERR!     at async ModuleLoader.fetchModule (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/rollup/dist/shared/rollup.js:19740:9)
ERR!     at async Promise.all (index 2)
ERR!     at async ModuleLoader.fetchStaticDependencies (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/rollup/dist/shared/rollup.js:19764:34)
ERR!     at async Promise.all (index 0)
ERR!     at async ModuleLoader.fetchModule (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/rollup/dist/shared/rollup.js:19740:9)
ERR!     at async Promise.all (index 0)
ERR!     at async ModuleLoader.fetchStaticDependencies (/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/rollup/dist/shared/rollup.js:19764:34)
ERR!     at async Promise.all (index 0) {
ERR!   watchFiles: [
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/iframe.html',
ERR!     '/virtual:/@storybook/builder-vite/vite-app.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/react/dist/esm/client/index.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/client-api/dist/esm/index.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/client-logger/dist/esm/index.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/.storybook/preview.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-docs/dist/esm/frameworks/common/config.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-docs/dist/esm/frameworks/react/config.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-links/dist/esm/preset/addDecorator.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-actions/dist/esm/preset/addDecorator.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-actions/dist/esm/preset/addArgs.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-backgrounds/dist/esm/preset/addDecorator.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-backgrounds/dist/esm/preset/addParameter.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-measure/dist/esm/preset/preview.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/storybook-addon-outline/dist/esm/preset/addDecorator.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/components/Button/Button.stories.tsx',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/react/dist/esm/client/preview/index.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/react/dist/esm/client/preview/types-6-3.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/client-api/dist/esm/client_api.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/client-api/dist/esm/decorators.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/client-api/dist/esm/parameters.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/client-api/dist/esm/story_store.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/client-api/dist/esm/config_api.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/client-api/dist/esm/pathToId.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/client-api/dist/esm/simulate-pageload.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/client-api/dist/esm/queryparams.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/client-api/dist/esm/filterArgTypes.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/client-api/dist/esm/hooks.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/client-api/dist/esm/types.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/client-api/dist/esm/inferControls.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/core-js/modules/es.array.concat.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/core-js/modules/es.set.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/core-js/modules/es.object.to-string.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/core-js/modules/es.string.iterator.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/core-js/modules/es.array.iterator.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/core-js/modules/web.dom-collections.iterator.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/core-js/modules/es.string.replace.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/core-js/modules/es.regexp.exec.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-docs/dist/esm/blocks/index.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-docs/dist/esm/frameworks/common/enhanceArgTypes.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-docs/dist/esm/frameworks/react/extractArgTypes.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-docs/dist/esm/lib/docgen/index.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-docs/dist/esm/frameworks/react/jsxDecorator.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-links/dist/esm/index.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-actions/dist/esm/index.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-actions/dist/esm/preset/addArgsHelpers.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-backgrounds/dist/esm/decorators/index.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/@storybook/addon-measure/dist/esm/withMeasure.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/storybook-addon-outline/dist/esm/withOutline.js',
ERR!     '/home/projects/storybook-vite-builder-client-logger-build-bug/node_modules/storybook-addon-outline/dist/esm/constants.js'
ERR!   ]
ERR! }
```
</details>

**Comment by eirslett** [comment]
I tried running it, but some files were missing (tsconfig, stitches.config...)

It looks like a lot is going on, can you try to reproduce the issue in a fresh repo, starting from

```
npm init @vitejs/app vite-react-app --template react && cd vite-react-app
npm install # or yarn
npx sb@next init --builder storybook-builder-vite && npm run storybook
```

and working in that direction?

**Comment by derekr** [comment]
Sorry about the missing files! Rushed getting a sample up. Will try to get a new repo up later today.

**Comment by derekr** [comment]
This issue is a red herring actually. I get this build error because of my overriding `process.env` and `globals` which breaks the build. I'm overriding those because I get a runtime error during dev (and a different runtime error when I try to serve the static build) relating to `process.env` not being defined which makes me think some server side code is being bundled and ran for some reason. Will try to reproduce that in a separate issue.

**Comment by eirslett** [comment]
Try `window.process = { env: {} }` maybe?

**Comment by derekr** [comment]
That helped a bit! Running in dev works and I can build without error, but get the same runtime error when serving the static build. Still planning on getting a reproduction of the dev runtime error that is causing server side code to get bundled.

This is the config I ended up with:

```js
async viteFinal(config, { configType }) {
    config.define = {
        'window.process': {
            env: {
                NODE_ENV: configType.toLowerCase(),
            },
        },
        'window.global': {},
    }
    config.esbuild = {
        jsxInject: `import React from 'react'`,
    }
    config.plugins.push(tsconfigPaths())
    return config
}
```

When I run `build-storybook` I get the following output:
```
❯ npm run build-storybook -w @myapp/goui                                                                                                          15:17:19

> @myapp/goui@1.0.0 build-storybook
> build-storybook -s public

info @storybook/react v6.3.4
info 
info => Cleaning outputDir: /Users/drk/dev/myapp/myapp-world/packages/goui/storybook-static
info => Copying static files: ./public => ./
info => Loading presets
info => Compiling manager..
vite v2.4.2 building for production...
transforming (349) ../../node_modules/@storybook/addon-links/node_modules/@storybook/addons/dist/esm/hooks.jsConflicting namespaces: "../../node_modules/@storybook/addon-essentials/node_modules/@storybook/client-api/dist/esm/index.js" re-exports "combineParameters" from both "../../node_modules/@storybook/addon-essentials/node_modules/@storybook/client-api/dist/esm/index.js" and "../../node_modules/@storybook/addon-essentials/node_modules/@storybook/client-api/dist/esm/parameters.js" (will be ignored)
transforming (920) ../../node_modules/lodash/_unicodeWords.jsConflicting namespaces: "../../node_modules/@storybook/addon-actions/node_modules/@storybook/client-api/dist/esm/index.js" re-exports "combineParameters" from both "../../node_modules/@storybook/addon-actions/node_modules/@storybook/client-api/dist/esm/index.js" and "../../node_modules/@storybook/addon-actions/node_modules/@storybook/client-api/dist/esm/parameters.js" (will be ignored)
transforming (961) node_modules/@storybook/addon-docs/node_modules/@storybook/components/dist/esm/syntaxhighlighter/formatter.jsConflicting namespaces: "node_modules/@storybook/addon-docs/node_modules/@storybook/client-api/dist/esm/index.js" re-exports "combineParameters" from both "node_modules/@storybook/addon-docs/node_modules/@storybook/client-api/dist/esm/index.js" and "node_modules/@storybook/addon-docs/node_modules/@storybook/client-api/dist/esm/parameters.js" (will be ignored)
transforming (1117) ../../node_modules/lodash/toNumber.jsConflicting namespaces: "../../node_modules/@storybook/client-api/dist/esm/index.js" re-exports "combineParameters" from both "../../node_modules/@storybook/client-api/dist/esm/index.js" and "../../node_modules/@storybook/client-api/dist/esm/parameters.js" (will be ignored)
transforming (1182) ../../node_modules/@storybook/react/node_modules/@storybook/client-logger/dist/esm/index.jsConflicting namespaces: "../../node_modules/@storybook/react/node_modules/@storybook/client-api/dist/esm/index.js" re-exports "combineParameters" from both "../../node_modules/@storybook/react/node_modules/@storybook/client-api/dist/esm/index.js" and "../../node_modules/@storybook/react/node_modules/@storybook/client-api/dist/esm/parameters.js" (will be ignored)
transforming (1373) ../../node_modules/lodash/_trimmedEndIndex.jsConflicting namespaces: "node_modules/@storybook/addon-docs/node_modules/@storybook/components/dist/esm/index.js" re-exports "components" from both "node_modules/@storybook/addon-docs/node_modules/@storybook/components/dist/esm/index.js" and "node_modules/@storybook/addon-docs/node_modules/@storybook/components/dist/esm/typography/DocumentFormatting.js" (will be ignored)
transforming (1454) ../../node_modules/entities/lib/maps/xml.jsonConflicting namespaces: "node_modules/@storybook/addon-docs/dist/esm/blocks/index.js" re-exports "SourceState" from both "node_modules/@storybook/addon-docs/dist/esm/blocks/Canvas.js" and "node_modules/@storybook/addon-docs/dist/esm/blocks/Source.js" (will be ignored)
transforming (1497) ../../node_modules/@popperjs/core/lib/utils/getFreshSideObject.jsConflicting namespaces: "../../node_modules/@storybook/addon-essentials/node_modules/@storybook/components/dist/esm/index.js" re-exports "components" from both "../../node_modules/@storybook/addon-essentials/node_modules/@storybook/components/dist/esm/index.js" and "../../node_modules/@storybook/addon-essentials/node_modules/@storybook/components/dist/esm/typography/DocumentFormatting.js" (will be ignored)
Conflicting namespaces: "../../node_modules/@storybook/addon-essentials/node_modules/@storybook/addon-docs/dist/esm/blocks/index.js" re-exports "SourceState" from both "../../node_modules/@storybook/addon-essentials/node_modules/@storybook/addon-docs/dist/esm/blocks/Canvas.js" and "../../node_modules/@storybook/addon-essentials/node_modules/@storybook/addon-docs/dist/esm/blocks/Source.js" (will be ignored)
Use of eval is strongly discouraged, as it poses security risks and may cause issues with minification
Use of eval is strongly discouraged, as it poses security risks and may cause issues with minification
Use of eval is strongly discouraged, as it poses security risks and may cause issues with minification
Use of eval is strongly discouraged, as it poses security risks and may cause issues with minification
✓ 1536 modules transformed.
rendering chunks (2)...Sourcemap is likely to be incorrect: a plugin (storybook-vite-inject-export-order-plugin) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help
Sourcemap is likely to be incorrect: a plugin (storybook-vite-inject-export-order-plugin) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help
Sourcemap is likely to be incorrect: a plugin (storybook-vite-inject-export-order-plugin) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help
Sourcemap is likely to be incorrect: a plugin (storybook-vite-inject-export-order-plugin) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help
Sourcemap is likely to be incorrect: a plugin (storybook-vite-inject-export-order-plugin) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help
Sourcemap is likely to be incorrect: a plugin (storybook-vite-inject-export-order-plugin) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help
Sourcemap is likely to be incorrect: a plugin (storybook-vite-inject-export-order-plugin) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help
Sourcemap is likely to be incorrect: a plugin (storybook-vite-inject-export-order-plugin) was used to transform files, but didn't generate a sourcemap for the transformation. Consult the plugin documentation for help
storybook-static/iframe.html                                 4.89kb
storybook-static/assets/syntaxhighlighter.e337683f.js        4.77kb / brotli: 1.86kb
storybook-static/assets/syntaxhighlighter.e337683f.js.map    18.74kb
storybook-static/assets/syntaxhighlighter.0e433746.js        4.77kb / brotli: 1.86kb
storybook-static/assets/syntaxhighlighter.0e433746.js.map    18.76kb
storybook-static/assets/iframe.e684e5dc.js                   13.31kb / brotli: 3.51kb
storybook-static/assets/iframe.e684e5dc.js.map               25.87kb
storybook-static/assets/WithTooltip.26491b2e.js              6.65kb / brotli: 2.04kb
storybook-static/assets/WithTooltip.26491b2e.js.map          23.01kb
storybook-static/assets/Color.ac13100a.js                    20.71kb / brotli: 7.08kb
storybook-static/assets/Color.ac13100a.js.map                77.31kb
storybook-static/assets/throttle.f99e26e6.js                 11.24kb / brotli: 3.66kb
storybook-static/assets/throttle.f99e26e6.js.map             37.89kb
storybook-static/assets/OverlayScrollbars.769011fa.js        2.16kb / brotli: 0.69kb
storybook-static/assets/OverlayScrollbars.769011fa.js.map    7.54kb
storybook-static/assets/GlobalScrollAreaStyles.7b232ff8.js   10.03kb / brotli: 1.94kb
storybook-static/assets/GlobalScrollAreaStyles.7b232ff8.js.map 19.87kb
storybook-static/assets/react-popper-tooltip.e31a6f1b.js     30.35kb / brotli: 8.72kb
storybook-static/assets/react-popper-tooltip.e31a6f1b.js.map 146.92kb
storybook-static/assets/WithTooltip.80f01cf9.js              6.65kb / brotli: 2.04kb
storybook-static/assets/WithTooltip.80f01cf9.js.map          22.98kb
storybook-static/assets/OverlayScrollbars.ec668bf2.js        2.16kb / brotli: 0.68kb
storybook-static/assets/OverlayScrollbars.ec668bf2.js.map    7.53kb
storybook-static/assets/prism-light.aac6f79f.js              58.20kb / brotli: 19.23kb
storybook-static/assets/prism-light.aac6f79f.js.map          215.51kb
storybook-static/assets/GlobalScrollAreaStyles.f587da16.js   10.03kb / brotli: 1.95kb
storybook-static/assets/GlobalScrollAreaStyles.f587da16.js.map 19.86kb
storybook-static/assets/Color.eb0f02c5.js                    20.71kb / brotli: 7.09kb
storybook-static/assets/Color.eb0f02c5.js.map                77.11kb
storybook-static/assets/OverlayScrollbars.111cf571.js        54.19kb / brotli: 20.37kb
storybook-static/assets/OverlayScrollbars.111cf571.js.map    476.60kb
storybook-static/assets/vendor.228c2e2b.js                   1560.45kb / brotli: skipped (large chunk)
storybook-static/assets/vendor.228c2e2b.js.map               4285.28kb

(!) Some chunks are larger than 500kb after minification. Consider:
- Using dynamic import() to code-split the application
- Use build.rollupOptions.output.manualChunks to improve chunking: https://rollupjs.org/guide/en/#outputmanualchunks
- Adjust chunk size limit for this warning via build.chunkSizeWarningLimit.
info => Manager built (32 s)
info => Output directory: /Users/drk/dev/myapp/myapp-world/packages/goui/storybook-static
```

And this error when serving:

![CleanShot 2021-07-19 at 15 21 00](https://user-images.githubusercontent.com/22249/126234777-18bca881-c360-4753-8a0d-340a25c8991f.png)

Which seems like a babel error so overriding those variables is messing with the storybook app itself

![CleanShot 2021-07-19 at 15 21 28](https://user-images.githubusercontent.com/22249/126234871-ec43e654-71c2-41df-a77c-3a3413df7578.png)

**Comment by derekr** [comment]
So I stopped pursuing debugging configuration and application setup when I couldn't reproduce the issue after copying my project exactly and installing dependencies fresh. I made a guess that it must be something with my dependency setup (I'm running a small multi-package repo using npm workspaces) and decided to blow away my package-lock.json file and reinstall dependencies from scratch and the error disappeared. My assumption is at some point while setting up the project initially I must have gotten in a funny state w/ my package-lock.json (like installing something incorrectly, moving dependencies around while learning a good setup for npm workspaces). Things worked for the most part, but bundling got tripped up.
